coderefs/coderefs.go

```diff
@@ -154,30 +168,31 @@ func runExtinctions(opts options.Options, matcher search.Matcher, branch ld.Bran
 	lookback := opts.Lookback
 	dryRun := opts.DryRun
 	if lookback > 0 {
-		missingFlags := []string{}
-		for flag, count := range branch.CountByFlag(matcher.Elements[0].Elements) {
-			if count == 0 {
-				missingFlags = append(missingFlags, flag)
-			}
+		var removedFlags []ld.ExtinctionRep
+		for i, project := range opts.Projects {
+			missingFlags := []string{}
+			for flag, count := range branch.CountByFlag(matcher.Elements[i].Elements, project.Key) {
```

Comment:
For how we generated and assign the matcher elements, there is one for each Project struct so we can use the index `i` here to look the corresponding values.